Packs/TAXIIServer/Integrations/TAXII2Server/TAXII2Server.py

```diff
@@ -0,0 +1,993 @@
+HTTP_401_UNAUTHORIZED = 401
+HTTP_404_NOT_FOUND = 404
+HTTP_406_NOT_ACCEPABLE = 406
+INTEGRATION_NAME: str = 'TAXII Server'
+API_ROOT = 'threatintel'
+APP: Flask = Flask('demisto-taxii2Z')
+NAMESPACE_URI = 'https://www.paloaltonetworks.com/cortex'
+MEDIA_TYPE_TAXII_ANY = 'application/taxii+json'
+MEDIA_TYPE_STIX_ANY = 'application/stix+json'
+MEDIA_TYPE_TAXII_V21 = 'application/taxii+json;version=2.1'
+MEDIA_TYPE_STIX_V21 = 'application/stix+json;version=2.1'
+MEDIA_TYPE_TAXII_V20 = 'application/vnd.oasis.taxii+json; version=2.0'
+MEDIA_TYPE_STIX_V20 = 'application/vnd.oasis.stix+json; version=2.0'
+ACCEPT_TYPE_ALL = '*/*'
+TAXII_VER_2_0 = '2.0'
+TAXII_VER_2_1 = '2.1'
+PAWN_UUID = uuid.uuid5(uuid.NAMESPACE_URL, 'https://www.paloaltonetworks.com')
+SCO_DET_ID_NAMESPACE = uuid.UUID('00abedb4-aa42-466c-9c01-fed23315a9b7')
+STIX_DATE_FORMAT = '%Y-%m-%dT%H:%M:%S.%fZ'
+
+XSOAR_TYPES_TO_STIX_SCO = {
+    FeedIndicatorType.CIDR: 'ipv4-addr',
+    FeedIndicatorType.DomainGlob: 'domain-name',
+    FeedIndicatorType.IPv6: 'ipv6-addr',
+    FeedIndicatorType.IPv6CIDR: 'ipv6-addr',
+    FeedIndicatorType.Account: 'user-account',
+    FeedIndicatorType.Domain: 'domain-name',
+    FeedIndicatorType.Email: 'email-addr',
+    FeedIndicatorType.IP: 'ipv4-addr',
+    FeedIndicatorType.Registry: 'windows-registry-key',
+    FeedIndicatorType.File: 'file',
+    FeedIndicatorType.URL: 'url',
+}
+
+XSOAR_TYPES_TO_STIX_SDO = {
+    ThreatIntel.ObjectsNames.ATTACK_PATTERN: 'attack-pattern',
+    ThreatIntel.ObjectsNames.CAMPAIGN: 'campaign',
+    ThreatIntel.ObjectsNames.COURSE_OF_ACTION: 'course-of-action',
+    ThreatIntel.ObjectsNames.INFRASTRUCTURE: 'infrastructure',
+    ThreatIntel.ObjectsNames.INTRUSION_SET: 'instruction-set',
+    ThreatIntel.ObjectsNames.REPORT: 'report',
+    ThreatIntel.ObjectsNames.THREAT_ACTOR: 'threat-actor',
+    ThreatIntel.ObjectsNames.TOOL: 'tool',
+    ThreatIntel.ObjectsNames.MALWARE: 'malware',
+    FeedIndicatorType.CVE: 'vulnerability',
+}
+
+STIX2_TYPES_TO_XSOAR = {
+    'campaign': ThreatIntel.ObjectsNames.CAMPAIGN,
+    'attack-pattern': ThreatIntel.ObjectsNames.ATTACK_PATTERN,
+    'report': ThreatIntel.ObjectsNames.REPORT,
+    'malware': ThreatIntel.ObjectsNames.MALWARE,
+    'course-of-action': ThreatIntel.ObjectsNames.COURSE_OF_ACTION,
+    'intrusion-set': ThreatIntel.ObjectsNames.INTRUSION_SET,
+    'tool': ThreatIntel.ObjectsNames.TOOL,
+    'threat-actor': ThreatIntel.ObjectsNames.THREAT_ACTOR,
+    'infrastructure': ThreatIntel.ObjectsNames.INFRASTRUCTURE,
+    'vulnerability': FeedIndicatorType.CVE,
+    "ipv4-addr": FeedIndicatorType.IP,
+    "ipv6-addr": FeedIndicatorType.IPv6,
+    "domain-name": [FeedIndicatorType.DomainGlob, FeedIndicatorType.Domain],
+    'user-account': FeedIndicatorType.Account,
+    'email-addr': FeedIndicatorType.Email,
+    "url": FeedIndicatorType.URL,
+    "file": FeedIndicatorType.File,
+    'windows-registry-key': FeedIndicatorType.Registry,
+}
+
+NGINX_TAXII2SERVER_CONF = '''
```

Comment:
didnt we say we'll take the conf from the api module and just update the cache keys ?